docs/tools/terraform/howto/config-postgresql-provider.rst

```diff
@@ -0,0 +1,80 @@
+Use PostgreSQL® provider with Aiven's PostgreSQL service
+#####################################################
+
+Aiven's Terraform provider is aimed for provisioning and performing basic service configuration, if additional configuration on the service is required there are other providers that can be used to perform the task. 
+This article shows how to use the `PostgreSQL® provider <https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs>`_ along with the ``Aiven Postgres service``.   
+
+
+Configure the required providers
+--------------------------------
+
+The new provider must be added to the ``required providers`` block in the Terraform code.
+
+1. This example shows how to add the ``PostgreSQL`` provider along with the ``Aiven`` provider
+
+.. code:: terraform
+
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">= 3.1"
+        }
+        postgresql = {
+          source  = "cyrilgdn/postgresql"
+          version = "1.16.0"
+        }        
+      }
+    }
+
+2. Then configure the provider with the corresponding information from the ``Aiven Postgres service``. 
+
+.. code:: terraform
+
+    provider "postgresql" {
+      host            = "pg-serivicename-projectname.aivencloud.com"
+      port            = 12691
+      database        = "defaultdb"
+      username        = "avnadmin"
+      password        = "postgres_password"
+      sslmode         = "require"
+      connect_timeout = 15
+    }
+
+Optionally, when the Aiven PG service is created in the same Terraform project the values required to configure the ``PostgreSQL`` provider can be passed using references to the resource, as shown in the code below:
```

Comment:
Optionally, when the Aiven for PostgreSQL service is created within the same Terraform project, the values required to configure the PostgreSQL provider can be passed using references to the resource, as shown in the code below: